src/build/build_marketclearance.jl

```diff
@@ -11,6 +11,12 @@ function build_marketclearance!(m, jm)
             push!(commodities, output.commodity)
         end
     end
+    for df in m._demands
+        for demand in df.demands
+            push!(commodities, demand.commodity)
+        end
+    end
+
 
```

Comment:
This is a fix to catch any commodity that ONLY goes directly from endowments to demands, and not through a production function.